runtime/minor_gc.c

```diff
@@ -341,16 +341,16 @@ static void oldify_one (void* st_v, value v, value *p)
     st->live_bytes += Bhsize_hd(hd);
     result = alloc_shared(sz, tag);
     for (i = 0; i < sz; i++) {
-      value curr = Op_val(v)[i];
-      Op_val (result)[i] = curr;
+      value curr = Field(v, i);
```

Comment:
Do we need the local variable `value curr`?

Reply:
You're right, we don't need it.